acceptance/Gemfile

```diff
@@ -11,8 +11,9 @@ def location_for(place, fake_version = nil)
 end
 
 gem "beaker-hostgenerator", *location_for(ENV['BEAKER_HOSTGENERATOR_VERSION'] || "~> 0.2")
-gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 2.8')
+gem "beaker", '~> 2.5'
```

Comment:
@zreichert how will the inclusion of `beaker-pe` be handled? Will the pending update https://github.com/puppetlabs/puppetdb-cli/pull/53 address this as you expect?

```
  if ENV["BEAKER_TYPE"] == 'pe'
    gem 'beaker-pe', '~> 0.9'
    gem 'scooter', '~> 3.2', :require => false
  end
```